carina-api/src/main/java/com/qaprosoft/carina/core/foundation/api/AbstractApiMethodV2.java

```diff
@@ -84,6 +88,35 @@ public AbstractApiMethodV2(String rqPath, String rsPath) {
         this(rqPath, rsPath, new Properties());
     }
 
+    public void replaceUrlPlaceholders() {
+        List<String> params = getParamsFromUrl();
+        for (String param : params) {
+            if (param != null && param.startsWith("config.env.")) {
+                String newParam = param.split("\\.")[2];
+                methodPath = methodPath.replace("${" + param + "}", Configuration.getEnvArg(newParam));
+            } else if (param != null && param.startsWith("config.")) {
+                String newParam = param.substring(7);
+                methodPath = methodPath.replace("${" + param + "}", R.CONFIG.get(newParam));
+            } else {
+                methodPath = methodPath.replace("${" + param + "}", "");
```

Comment:
it will replace all other placeholders with nothing which is wrong
we don't need this else at all